build-scripts/build_xccdf.py

```diff
@@ -49,11 +54,70 @@ def parse_args():
         dest="build_ovals_dir",
         help="Directory to store OVAL document for each rule.",
     )
-    parser.add_argument("--resolved-base",
-                        help="To which directory to put processed rule/group/value YAMLs.")
+    parser.add_argument(
+        "--resolved-base",
+        help="To which directory to put processed rule/group/value YAMLs."
+    )
+    parser.add_argument(
+        "--per-profile-dir",
+        help="Directory to store XCCDF, OVAL, OCIL, per profile. (off: to disable)"
```

Comment:
Per profile? I'm confused. It seems like they're per rule. Can you clarify?

Reply:
The whole process of creating a thin datastream is that a profile is generated for each rule. (In case only one rule is specified is created one profile) This is done in the `compile_all.py` file.  When the `--thin` flag is set, XCCDF, OVAL and OCIL components need to be generated for each rule. Since each rule has its own profile the generation is done by profiles. (In case the one rule is specified `--per-profile-dir` flag is disabled). Also, this can be used sometime in the future when we want to build a datastream with only one profile.